platforms/software/build-init/src/integTest/groovy/org/gradle/buildinit/plugins/BuildInitPluginIntegrationTest.groovy

```diff
@@ -219,6 +221,17 @@ class BuildInitPluginIntegrationTest extends AbstractInitIntegrationSpec {
         scriptDsl << ScriptDslFixture.SCRIPT_DSLS
     }
 
+    def "proper links"() {
+
+        when:
+        succeeds('init', '--type', 'java-application', '--dsl', GROOVY.toString().toLowerCase())
+
+
+        then:
+
+        targetDir.file("settings.gradle").assertContents(containsString(userManual("multi_project_builds").getUrl()))
+    }
```

Comment:
💭 Does this actually require a test? If yes, can we fix the whitespace? 🙃

Reply:
If this would have been there before, I couldn't have broken it. (not sure I got the tenses right here ;) )
So I think it's nice to have.